docs/source/getting-started/path-variables.rst

```diff
@@ -24,48 +24,48 @@ If you have downloaded and installed the SDK using our `Windows installer <https
 
 Mac OS
 ******
-We'll explain here how to set up ``JAVA_HOME`` and ``PATH`` variables on Mac OS with ``zsh`` shell.
-If you are using ``bash`` all of the instructions are quite similar, except that you will be doing all of the changes
-in the ``.bash_profile`` file.
 
-Setting the JAVA_HOME variable
-==============================
-Run the following command in your terminal::
+First, you need to figure out whether you are running Bash or zsh. To do that, open a Terminal and run::
 
-        echo 'export JAVA_HOME="$(/usr/libexec/java_home)"' >> ~/.zprofile
+        echo $SHELL - $0
 
-Setting the PATH variable
-=========================
-The installer will ask you and set the ``PATH`` variable for you. If you want to set your ``PATH`` variable
-manually instead, run the following command in your terminal::
+This should return either ``/bin/bash - bash``, in which case you are running
+Bash, or ``/bin/zsh - -zsh``, in which case you are running zsh. We provide
```

Comment:
IME this is slightly inaccurate. I get `/bin/bash - -bash` if Terminal is in bash mode, and `/bin/bash - bash` if I start up `bash` manually. And `/bin/zsh - zsh` if I start up `zsh` manually. Basically the presence of the extra dash (`-`) seems to depend on if you're running your default shell. So my suggestion is to put the extra dash in there in both cases:  

```suggestion
This should return either ``/bin/bash - -bash``, in which case you are running
Bash, or ``/bin/zsh - -zsh``, in which case you are running zsh. We provide
```

Reply:
I did not realize this was so fragile, so instead I've just removed `$0`. Do you see any issue with that approach? Unless you're doing something special already (in which case these instructions are probably not for you), `$SHELL` should be correct, I think.

Reply:
I think that should work! Thanks :)